docs/user-manual/design/fpp-json-dict.md

```diff
@@ -729,39 +789,130 @@ product record Record1: U32 id 0x102
 
 Example FPP model with JSON representation:
 ```
-@ Container 0
-@ Implied id is 0x100
-product container Container0
+module M {
+
+  active component Component1 { 
+
+    @ Container 0
+    product container Container0 id 0x07
+
+    @ Container 1
+    product container Container1 id 0x08
+
+    @ Container 2
+    product container Container2 id 0x09 default priority 10
+
+  }
 
-@ Container 1
-product container Container1 id 0x102
+  instance C1: Component1 base id 0x100 \
+    queue size 10
+
+  topology T {
+    instance C1
+  }
+
+}
 
-@ Container 2
-@ Implied id is 0x103
-product container Container2 default priority 10
 ```
 
 ```json
 [
     {
-       "name": "M1.Container0",
-       "annotation": "Container 0\nImplied id is 0x100",
-       "id": 256,
+       "name": "M.C1.Container0",
+       "annotation": "Container 0",
+       "id": 263,
     },
     {
-        "name": "M1.Container1",
+        "name": "M.C1.Container1",
         "annotation": "Container 1",
-        "id": 258,
+        "id": 264,
     },
     {
-        "name": "M1.Container2",
-        "annotation": "Container 2\nImplied id is 0x103",
-        "id": 3,
+        "name": "M.C1.Container2",
+        "annotation": "Container 2",
+        "id": 265,
         "defaultPriority": 259
     }
 ]
 ```
 
+## Telemetry Packet Sets
+
+### Telemetry Packets
+| Field | Description | Options | Required |
```

Comment:
Can we add a newline between the heading and the table here? This used to break the rendering on the old website - not sure about the new one, but might as well be careful.

Reply:
And in the below headings too

Reply:
Resolved in my latest commit!